docs/design/daemon.md

```diff
@@ -0,0 +1,128 @@
+  - Using the pod's node name field, Daemon controllers can be restricted to operate on a specified node.
+  - The spec for pod templates that run with the Daemon Controller is the same as the spec for pod templates that run with the Replication Controller, except there will not be a ‘replicas’ field (exactly 1 daemon pod will be launched per node).
+  - We will not guarantee that daemon pods show up on nodes before regular pods - run ordering is out of scope for this controller. 
+  - The initial implementation of Daemon Controller does not guarantee that Daemon pods show up on nodes (for example because of resource limitations of the node), but makes a best effort to launch Daemon pods (like Replication Controllers do with pods). Subsequent revisions might ensure that Daemon pods show up on nodes, pushing out other pods if necessary.
+  - A daemon controller named “foo” adds a “controller: foo” annotation to all the pods that it creates
+  - YAML example:
+```YAML
+  apiVersion: v1
+  kind: Daemon
+  metadata:
+    labels:
+      name: datastore
+    name: datastore
+  spec:
+    template:
+      metadata:
+        labels:
+          name: datastore-shard
+      spec:
+        node-selector: 
+          name: datastore-node
+        containers:
+          name: datastore-shard
+          image: kubernetes/sharded
+          ports:
+            - containerPort: 9042
+              name: main
+```
+  - commands that get info
+    - get (e.g. kubectl get dc)
+    - describe
+  - Modifiers
+    - delete
+    - stop: first we turn down all the pods controller by the daemon (by setting the nodeName to a non-existed name). Then we turn down the daemon controller.
+    - label
+    - update
+    - Daemon controllers have labels, so you could, for example, list all daemon controllers with a certain label (the same way you would for a Replication Controller).
+  - In general, for all the supported features like get, describe, update, etc, the Daemon Controller works in a similar way to the Replication Controller. However, note that the Daemon Controller and the Replication Controller are different constructs.
+
+### Persisting Pods
+  - Ordinary health checks specified in the pod template work to keep pods created by a Daemon Controller running.
+  - If a daemon pod is killed or stopped, the daemon controller will create a new replica of the daemon pod on the node.
+
+### Cluster Mutations
+  - When a new node is added to the cluster the daemon controller starts the daemon on the node (if the node’s labels match the user-specified selectors). This is a big advantage of the Daemon Controller compared to alternative ways of launching daemons and configuring clusters.
```

Comment:
remove "This is a big advantage of the Daemon Controller compared to alternative ways of launching daemons and configuring clusters".  

Otherwise you would need to provide a sane contrast.  As opposed to using ...